src/Security/Handler/AclSecurityHandlerInterface.php

```diff
@@ -67,23 +68,31 @@ public function findObjectAcls(\Traversable $oids, array $sids = []);
 
     /**
      * Add an object owner ACE to the object ACL.
+     *
+     * NEXT_MAJOR: change signature to `addObjectOwner(MutableAclInterface $acl, ?UserSecurityIdentity $securityIdentity = null): void`.
      */
     public function addObjectOwner(AclInterface $acl, ?UserSecurityIdentity $securityIdentity = null);
 
     /**
      * Add the object class ACE's to the object ACL.
+     *
+     * NEXT_MAJOR: change signature to `addObjectClassAces(MutableAclInterface $acl, array $roleInformation = []): void`.
      */
     public function addObjectClassAces(AclInterface $acl, array $roleInformation = []);
 
     /**
      * Create an object ACL.
      *
-     * @return AclInterface
+     * NEXT_MAJOR: change signature to `createAcl(ObjectIdentityInterface $objectIdentity): MutableAclInterface`
+     *
+     * @return MutableAclInterface
      */
     public function createAcl(ObjectIdentityInterface $objectIdentity);
 
     /**
      * Update the ACL.
+     *
+     * NEXT_MAJOR: change signature to `updateAcl(MutableAclInterface $acl): void`
```

Comment:
You can already add this information with phpdoc right?

Reply:
done